### PR TITLE
Update URL for 2FA

### PR DESCRIPTION
**Pull request by jkerr321**

### CODE REVIEW

**Review by dapenguin — APPROVED** [review]
LGTM 👍

**Review by leggsimon — APPROVED** [review]
👍

**Comment by leggsimon** [comment]
Maybe don't merge it just yet actually

File: lib/uriConstructor.js [hunk 1]
```diff
@@ -1,11 +1,13 @@
+const { stringify } = require('querystring');
 const logger = require('./logger');
-const config = require('./config');
-const loginUrl = config.LOGIN_URL;
+const loginUrl = 'https://beta-accounts.ft.com/login?';
```
Comment (Thurst-ft) [inline lib/uriConstructor.js]: Why is this `beta-accounts`?

**Review by leggsimon — COMMENTED** [review]
Similar to my comments on https://github.com/Financial-Times/kat-client-proxies/pull/47

I think both the `getUserId` and `redirectUri` functions could be refactored a bit to reduce some unnecessary duplication.

As far as I can tell in the `getUserId` function, the only things that change depending on the `kat2fa` flag is which cookie we get (`'FTSession_s'` over `'FTSession'`) and passing the flags to `sessionClient.verify`. In this case I don't think it requires a whole `if...else` block but rather just change which cookie we get, we can still pass the flags in both instances. This way instead of having a whole `if...else` we could just have 

```js
const sessionToken = (res.locals && res.locals.flags && res.locals.flags.kat2fa)
	? cookieHandler.get(req, res, 'FTSession_s')
	: cookieHandler.get(req, res, 'FTSession');
```

File: lib/middleware/verifySession.js [hunk 2]
```diff
@@ -6,21 +6,33 @@ const sessionClient = require('@financial-times/kat-client-proxies').sessionClie
 
 function getUserId (req, res, next) {
 	const operation = 'verifySession.getUseId';
```
Comment (leggsimon) [inline lib/middleware/verifySession.js]: do you mind just fixing this typo

```diff
- getUseId
+ getUserId
```

**Pull request merged**